release-notes/major12/12.1.2/valtimo-frontend-libraries.md

```diff
@@ -0,0 +1,73 @@
+# Frontend libraries 12.1.2
+
+## New Features
+
+The following features were added:
+
+* **New feature1**
+
+  Description of the new feature goes here.
+  Also link to the page that expains the feature in greater detail.
+
+* **New feature2**
+
+  Description of the new feature goes here.
+  Also link to the page that expains the feature in greater detail.
+
+
+## Bugfixes
+
+The following bugs were fixed:
+
+* **User cannot click 'save' in Formio sandbox**
+
+  In some cases, the user could not click 'save' in the Formio sandbox.  
+
+* **Many duplicate & cancelled requests to the API**
+
+  When navigating through the application, 
```

Comment:
Almost reads like a haiku with the linebreaks like this

Reply:
That's not visible when rendered though. I just kept it withing the 80-isch characters.